test/config.bats

```diff
@@ -11,6 +11,12 @@ function teardown() {
 	cleanup_test
 }
 
+@test "default config should be empty" {
+	setup_crio
+	output=$("$CRIO_BINARY_PATH" config "|sed 's/#.*//' | tr -d '\n'")
+	[[ "$output" == *"[crio]"* ]]
```

Comment:
I believe this will fail now that we're not commenting the headers

Reply:
or rather, it should fail and I am actually not sure it does because we use a wildcard * at the end (thus allowing for anything after [crio]

I thiink this test case needs an update regardless

Reply:
should it check to find all of the headers?

Reply:
yeah, or it could check there are no lines that don't start with `[`

Reply:
that may be easier to maintain because we may add headers in the future